src/main/java/org/simulator/fba/FluxBalanceAnalysis.java

```diff
@@ -50,433 +50,419 @@
+   */
+  public Map<String, Double> getSolution() {
+    Map<String, Double> result = new HashMap<>();
+    result.put(actObjFunc, getObjectiveValue());
+    for (Map.Entry<String, Integer> mapElement : reaction2Index.entrySet()) {
+      result.put(mapElement.getKey(), solution[mapElement.getValue()]);
+    }
+    return result;
+  }
+
+  /**
+   * Determines the stoichiometry value of a given {@link SpeciesReference}.
+   * This might involve the evaluation of a
+   * {@link org.sbml.jsbml.StoichiometryMath} or needs to lookup the current
+   * stoichiometry value if it has been changed by an initial assignment.
+   *
+   * @param specRef the {@link SpeciesReference} whose stoichiometry value needs to be
+   *                determined
+   * @return a double value indicating the stoichiometry value of the given
+   * {@link SpeciesReference}. This value can be directly specified by
+   * the element, or needs to be calculated from its
+   * {@link org.sbml.jsbml.StoichiometryMath} or through an
+   * {@link InitialAssignment}.
+   * @throws ModelOverdeterminedException if the model cannot be solved because too many equations over
+   *                                      determine its solution space (this can happen if algebraic rules
+   *                                      are used in the model).
+   * @throws SBMLException                if the model has an invalid structure.
+   */
+  private double stoichiometry(SpeciesReference specRef)
+      throws SBMLException, ModelOverdeterminedException {
+    if ((interpreter != null) && specRef.isSetId()) {
+      // There could be an initial assignment that has changed the value of this speciesReference.
+      return interpreter.getCurrentStoichiometry(specRef.getId());
+    } else if (specRef.isSetStoichiometry()) {
+      return specRef.getStoichiometry();
+    } else {
+      if (interpreter == null) {
+        interpreter = new SBMLinterpreter(specRef.getModel());
+      }
+      if (specRef.isSetStoichiometryMath()) {
+        return ((ASTNodeValue) specRef.getStoichiometryMath().getMath().getUserObject(SBMLinterpreter.TEMP_VALUE)).compileDouble(interpreter.getCurrentTime(), 0d);
+      } else if (specRef.isSetId()) {
+        // Is there an initial assignment?
+        interpreter.getCurrentStoichiometry(specRef.getId());
+      } else {
+        throw new SBMLException("Could not calculate the stoichiometry for a species reference because it was lacking an identifier.");
+      }
+    }
+    return Double.NaN;
+  }
+
+  /**
+   * Gets the value of the EPSILON
+   *
+   * @return the epsilon value
+   */
+  public double getEpsilon() {
+    return eps;
+  }
+
+  /**
+   * Set the value of the EPSILON specific to a particular FBC instance
+   *
+   * @param eps
+   */
+  public void setEpsilon(double eps) {
+    this.eps = eps;
+  }
+
+  /**
+   * Gets the id of the active objective function
+   *
+   * @return
+   */
+  public String getActObjFunc() {
+    return actObjFunc;
```

Comment:
Better name is `activeObjective`. Always write words out. There is code completion which makes larger names easy to use. Also rename functions to `getActiveObjective`.

Reply:
Updated!